misk-hibernate/src/main/kotlin/misk/jdbc/DataSourceConfig.kt

```diff
@@ -103,24 +124,31 @@ data class DataSourceConfig(
       DataSourceType.VITESS -> {
         var queryParams = ""
         var useSSL = false
+
+        // NOTE(nb): still support url properties in Vitess for backwards compatibility.
```

Comment:
nb(nb)

Reply:
IMO it's a little confusing to support both, do we plan on migrating clients and phasing support out?

Reply:
Yeah, the plan is to migrate folks and phasing out the use of the url, but we need to support both until that day comes.